src/NServiceBus.Gateway.AcceptanceTests/NServiceBus.Gateway.AcceptanceTests.csproj

```diff
@@ -19,6 +19,23 @@
     <PackageReference Include="NServiceBus.Callbacks" Version="3.0.0-beta0003" />
     <PackageReference Include="NUnit" Version="3.7.1" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.8.0-alpha1" />
+    <PackageReference Include="Particular.Packaging" Version="0.1.0" PrivateAssets="All" />
```

Comment:
The rest of the package references either need `PrivateAssets="All"`, or a proper version range for the dependencies that should be on the package.

Private:
- Microsoft.NET.Test.Sdk
- NUnit
- NUnit3TestAdapter

Range:
- NServiceBus.AcceptanceTesting:  `[7.0.0-beta0012, 8.0.0)`
- NServiceBus.Callbacks: `[3.0.0-beta0003, 4.0.0)`

Reply:
NServiceBus.AcceptanceTesting brings in a transitive NUnit dependency, so this project could drop the NUnit dependency.

Alternatively, if this project bumped up the version of NUnit it uses, you could argue that the package would then need a public NUnit as well, to ensure the proper minimum version is picked up.